Tests/Java2SwiftTests/Java2SwiftTests.swift

```diff
@@ -84,6 +103,7 @@ func assertTranslatedClass<JavaClassType: AnyJavaObject>(
     """
 
   for expectedChunk in expectedChunks {
+    print(swiftFileText)
```

Comment:
stray print?

Reply:
oops, it made it much easier for testing that what came out actually built ... but removed it now